clang-tools-extra/clang-tidy/bugprone/SizeofExpressionCheck.cpp

```diff
@@ -342,31 +342,33 @@ void SizeofExpressionCheck::check(const MatchFinder::MatchResult &Result) {
 
     if (DenominatorSize > CharUnits::Zero() &&
         !NumeratorSize.isMultipleOf(DenominatorSize)) {
-      diag(E->getOperatorLoc(), "suspicious usage of 'sizeof(...)/sizeof(...)';"
+      diag(E->getOperatorLoc(), "suspicious usage of 'sizeof(...)/sizeof(...)':"
                                 " numerator is not a multiple of denominator")
           << E->getLHS()->getSourceRange() << E->getRHS()->getSourceRange();
     } else if (ElementSize > CharUnits::Zero() &&
                DenominatorSize > CharUnits::Zero() &&
                ElementSize != DenominatorSize) {
-      diag(E->getOperatorLoc(), "suspicious usage of 'sizeof(...)/sizeof(...)';"
-                                " numerator is not a multiple of denominator")
+      diag(E->getOperatorLoc(),
+           "suspicious usage of 'sizeof(array)/sizeof(...)':"
```

Comment:
```suggestion
           "suspicious usage of 'sizeof(...)/sizeof(...)':"
```

Maybe it would be better to keep the scheme here?

Reply:
What do you think "division of two sizeof expression" instead of "sizeof(...)/sizeof(...)"

Reply:
It would be slightly better in isolation, but all the other diagnostics look like e.g. `suspicious usage of 'sizeof(..., ...)'` or `suspicious usage of 'sizeof(this)'; did you mean 'sizeof(*this)'` so for the sake of consistency I'd like to leave this aspect of the messages unchanged.